h2o-algos/src/main/java/hex/tree/DHistogram.java

```diff
@@ -302,10 +302,14 @@ public void init(final double[] vals) {
     if (_histoType==SharedTreeModel.SharedTreeParameters.HistogramType.Random) {
       // every node makes the same split points
       Random rng = RandomUtils.getRNG((Double.doubleToRawLongBits(((_step+0.324)*_min+8.3425)+89.342*_maxEx) + 0xDECAF*_nbin + 0xC0FFEE*_isInt + _seed));
-      assert _nbin > 1;
-      _splitPts = makeRandomSplitPoints(_nbin, rng);
-    }
-    else if (_histoType== SharedTreeModel.SharedTreeParameters.HistogramType.QuantilesGlobal) {
+      assert(_nbin>1);
```

Comment:
this looks strange, this looks like you reverted code I wrote

Reply:
Maybe I did rebase incorrectly... Will check.